AudioSensei/Program.cs

```diff
@@ -26,7 +27,7 @@ namespace AudioSensei
 {
     class Program
     {
-        public static event Action Exit;
+        [NotNull] public static event Action Exit;
```

Comment:
It shouldn't be in the same line

Reply:
This isn't even correct

Reply:
I wasn't sure what formatting conventions you use
```suggestion
        [NotNull]
        public static event Action Exit;
```

Reply:
But it can be null though

Reply:
```suggestion
        [CanBeNull]
        public static event Action Exit;
```